### ISSUE TITLE
Server crash

### ISSUE THREAD

**Issue by ab9rf** [supermassimo/DynamicTrees-ExC#2]
Mod crashes with `java.lang.NoClassDefFoundError` during server startup when used in a standalone server. This happens while trying to instantiate a client-only class (`net.minecraft.client.renderer.color.IBlockColor`) so that suggests to me that you have client-only code being invoked from a common method, making this mod unusable with a standalone server.

Relevant segment of traceback:
```maxhyper.dynamictreesexc.ModContent.registerBlocks(ModContent.java:78) ~[ModContent.class:?]```

**Comment by supermassimo** [comment]
Will take a look, should be fixed by next update.

**Comment by Invidious** [comment]
Waiting for this :)

**Comment by LoneDev6** [comment]
```
Time: 17/04/20 19.35
Description: Exception in server tick loop

java.lang.NoClassDefFoundError: net/minecraft/client/renderer/block/statemap/IStateMapper
	at maxhyper.dynamictreesexc.ModContent.registerBlocks(ModContent.java:78)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_98_ModContent_registerBlocks_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:845)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630)
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99)
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.block.statemap.IStateMapper
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 12 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 14 more

```

**Comment by beamthelaser** [comment]
> java.lang.NoClassDefFoundError: net/minecraft/client/renderer/block/statemap/IStateMapper
	at maxhyper.dynamictreesexc.ModContent.registerBlocks(ModContent.java:78) ~[ModContent.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_234_ModContent_registerBlocks_Register.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144) ~[EventBus$1.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) ~[EventBus.class:?]
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:845) ~[GameData.class:?]
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630) ~[Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333) ~[FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125) ~[nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_251]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.block.statemap.IStateMapper
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_251]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_251]
	... 12 more
Caused by: java.lang.NullPointerException
[12:46:08] [Server thread/ERROR] [net.minecraft.server.MinecraftServer]: This crash report has been saved to: D:\Program (x86)\Minecraft Forge Server\.\crash-reports\crash-2020-05-06_12.46.08-server.txt
[12:46:08] [Server thread/INFO] [net.minecraft.server.MinecraftServer]: Stopping server
[12:46:08] [Server thread/INFO] [net.minecraft.server.MinecraftServer]: Saving worlds
[12:46:08] [Server thread/INFO] [FML]: The state engine was in incorrect state INITIALIZATION and forced into state SERVER_STOPPED. Errors may have been discarded.
[12:46:08] [Server Shutdown Thread/INFO] [net.minecraft.server.MinecraftServer]: Stopping server

Similar issue to LoneDev6

**Comment by ztx-lyghters** [comment]
> [Server thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.event.RegistryEvent$Register@6990aed4:
java.lang.NoClassDefFoundError: net/minecraft/client/renderer/block/statemap/IStateMapper
	at maxhyper.dynamictreesexc.ModContent.registerBlocks(ModContent.java:78) ~[ModContent.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_241_ModContent_registerBlocks_Register.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144) ~[EventBus$1.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:845) [GameData.class:?]
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630) [Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99) [FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333) [FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125) [nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_251]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.block.statemap.IStateMapper
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_251]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_251]
	... 12 more
Caused by: java.lang.NullPointerException

Trying to run forge 1.12.2-14.23.5.2854 server

**Comment by supermassimo** [comment]
Fixed in [Update 1.0](https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-integrated-dynamics/files)
